webapp/src/services/config.ts

```diff
@@ -1,4 +1,11 @@
+const APP_ID = 'f964e3a3-7c85-4819-9dab-78ae54d5f9ba';
+
+const SERVER = 'http://127.0.0.1:6868';
```

Comment:
on peut mettre ces valeurs là dans le `.env` surtout l'app_id car elle change souvent d'une personne à l'autre etc suivant les builds
+ niveau sécu c'est mieux quand même

```suggestion
const APP_ID = process.env.API_APP_ID;

const SERVER = process.env.API_BASE_URL;
```